package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "scripts": {
     "lint": "eslint src/ test/",
+    "extract-from-assets": "node scripts/get_assets_properties.js > src/assets_properties.js",
```

Comment:
To fix the error below, have you tried with `node -r esm scripts/get_assets_properties.js > src/assets_properties.js`?

Reply:
the error is now:
```
/home/fredj/code/ngm/node_modules/cesium/Source/Core/Rectangle.js:1
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/fredj/code/ngm/node_modules/cesium/Source/Core/Rectangle.js
require() of ES modules is not supported.
require() of /home/fredj/code/ngm/node_modules/cesium/Source/Core/Rectangle.js from /home/fredj/code/ngm/src/constants.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename Rectangle.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /home/fredj/code/ngm/node_modules/cesium/package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:13) {
  code: 'ERR_REQUIRE_ESM'
}
```

Reply:
@fredj, I pushed a workaround to fix this. You can continue from there.

Reply:
Thanks ! I started with the same idea yesterday but didn't finished

Reply:
It works but the `export` keyword is not supported by node:
```
export const LAS_POINT_CLOUD_STYLE = {
^^^^^^

SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:1053:16)
    at Module._compile (internal/modules/cjs/loader.js:1101:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at ModuleWrap.<anonymous> (internal/modules/esm/translators.js:155:15)
    at ModuleJob.run (internal/modules/esm/module_job.js:139:37)
    at async Loader.import (internal/modules/esm/loader.js:179:24)
```